code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -0,0 +1,148 @@
+/obj/item/storage/box/bs_kit
+	name = "\improper Blackshield Kit"
+	desc = "A standard kit."
+	icon_state = "box"
+
+// All-in-one kit
+/obj/item/storage/box/bs_kit/default
+	name = "\improper Default Blackshield Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/boltgun(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/gun/projectile/colt/ten/dark(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/clothing/accessory/holster/leg(src)
+		new /obj/item/clothing/suit/armor/platecarrier/militia(src)
+		new /obj/item/clothing/head/helmet/ballistic/militia(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+// Primary Weapon Kits
+/obj/item/storage/box/bs_kit/mosin
+	name = "\improper Mosin Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/boltgun(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/sts
+	name = "\improper STS Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/automatic/sts/rifle/blackshield(src)
```

Comment:
This is the best STS in the game, theirs only meant to be 4 that requires customs do to being hand made

Reply:
All I knew is that it was a blackshield-themed gun.

Reply:
It was replaced by the normal STS